tests/unit/components/UI/Navigation/Paginator.spec.js

```diff
@@ -10,4 +10,47 @@ describe("Paginator.vue", () => {
     });
     expect(wrapper.get('[data-test="totalItems"]').text()).toBe("10");
   });
+
+  it("should trigger correctly on click", async () => {
```

Comment:
A little more descriptive name would be very helpful. Like I'm not sure which click are we talking about?
There are next/prev buttons plus page buttons as well. Just mentioning ki which button clicks are we testing for should help a lot